### DISCUSSION TITLE
zh/best-practices/nix-path-and-flake-registry

### DISCUSSION THREAD

**Discussion by giscus** [ryan4yin/nixos-and-flakes-book#171] [Announcements]
# zh/best-practices/nix-path-and-flake-registry

一份非官方的新手指南

https://nixos-and-flakes.thiscute.world/zh/best-practices/nix-path-and-flake-registry

**Comment by segment-tree** [comment]
(提醒:如果 "error: attribute 'nixpkgs' missing"可以把nixpkgs换成inputs.nixpkgs.)

**Reply by SaltyKitkat** [reply]
我也遇到了这个错误，请问需要在哪里将哪个nixpkgs换成inputs.nixpkgs呢
